staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter.go

```diff
@@ -29,19 +29,18 @@ import (
 	"sync/atomic"
 
 	apiequality "k8s.io/apimachinery/pkg/api/equality"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/diff"
 	"k8s.io/apimachinery/pkg/util/json"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
 	"github.com/golang/glog"
 )
 
-// Converter is an interface for converting between runtime.Object
+// Converter is an interface for converting between interface{}
```

Comment:
There was zero type safety in this code before - runtime.Object isn't type safety, and this code doesn't know anything about runtime objects.